share/man/man1/urlwatch.1

```diff
@@ -71,13 +71,13 @@ list supported jobs/filters/reporters
 remove old cache entries
 .SH "FILES"
 .TP
-.B ~/.urlwatch/urls.yaml
+.B ~/.config/urlwatch/urls.yaml
```

Comment:
Well, this should mention `$XDG_CONFIG_DIR`.

Reply:
Okido replaced it with XDG_CONFIG_DIR :)